imgee/templates/show_label.html.jinja2

```diff
@@ -4,9 +4,9 @@
 {% block maincontent %}
   <section class="show-label">
     <div class="mui--clearfix">
-      <div class="mui--pull-left"><span class="mui-text--title">Images with label</span> <span class="editable_label mui--d-inlineblock" id="{{label.id}}">{{label.name}}</span></div>
+      <div class="mui--pull-left"><span class="mui-text--title">{% trans %}Images with label{% endtrans %}</span> <span class="editable_label mui--d-inlineblock" id="{{label.id}}">{{label.name}}</span></div>
```

Comment:
This is an example of the label being part of a sentence, and yet being (necessarily) outside the trans tag. The correct fix is to remove the text and use an icon representing a label. @vidya-ram you'll have to pick one.

Reply:
The labels are user defined and font awesome might not have options available for all and naming conventions will be different too.

Reply:
The span with `editable_label` class has a JS function attached to it as well. It opens form to edit the label on click